aws_s3_policies/aws_s3_bucket_public_read.py

```diff
@@ -6,9 +6,10 @@
 
 
 def policy(resource):
-    for grant in resource['Grants']:
-        if grant['Grantee']['URI'] in GRANTEES and grant[
-                'Permission'] in PERMISSIONS:
-            return False
+    if 'Grants' in resource and resource['Grants'] is not None:
```

Comment:
Nit: the built-in `dict.get` should be used in place of `dict[key]` when possible

This applies to a lot of the python files in this PR